### DISCUSSION TITLE
2/5

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#85] [Announcements]
# 2/5

SICP Python 中文翻译

https://composingprograms.netlify.app/2/5

**Comment by zhangeecs** [comment]
<2.5.2 类的定义
在 class 的 <suite> 中 def 或赋值语句中绑定的<如何>名称都会创建或修改类的属性。>
应该是<任何>吧

**Reply by mancuoj** [reply]
已修改♥
